nxdrive/data/qml/NuxeoSwitch.qml

```diff
@@ -34,7 +34,7 @@ Switch {
             width: roundSize
             height: roundSize
             radius: size
-            color: control.checked ? (control.down ? lighterGray : "white") : uncheckedColor
+            color: control.checked ? (control.down ? uiBackground : "white") : uncheckedColor
```

Comment:
Here we should have something else than "white", right? We should use a predefinied color.